default/stringtables/en.txt

```diff
@@ -10448,91 +10448,92 @@ SHP_WEAPON_1_2
 
 SHP_WEAPON_1_3
 Mass Driver 3
 
 SHP_WEAPON_1_3_DESC
-Improves Mass Driver by two levels, increasing strength by 2.
+Upgrades [[shippart SR_WEAPON_1_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 5.
 
 SHP_WEAPON_1_4
 Mass Driver 4
 
 SHP_WEAPON_1_4_DESC
-Improves Mass Driver by three levels, increasing strength by 3.
+The ultimate refinement of the Mass Driver. Upgrades [[shippart SR_WEAPON_1_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 6.
 
 SHP_WEAPON_2_1
 Laser Weapons
 
 SHP_WEAPON_2_1_DESC
-Unlocks the Laser, a more powerful ship's weapon than the Mass Driver.
+Unlocks the [[shippart SR_WEAPON_2_1]], a more powerful ship's weapon than the [[shippart SR_WEAPON_1_1]].
 
 SHP_WEAPON_2_2
 Laser 2
 
 SHP_WEAPON_2_2_DESC
-Improves Laser weapons by one level, increasing strength by 2.
+Upgrades [[shippart SR_WEAPON_2_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 7.
 
 SHP_WEAPON_2_3
 Laser 3
 
 SHP_WEAPON_2_3_DESC
-Improves Laser weapons by two levels, increasing strength by 4.
+Upgrades [[shippart SR_WEAPON_2_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 9.
 
 SHP_WEAPON_2_4
 Laser 4
 
 SHP_WEAPON_2_4_DESC
-Improves Laser weapons by three levels, increasing strength by 6.
+The ultimate refinement of the Laser weapon. Upgrades [[shippart SR_WEAPON_2_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 11.
 
 SHP_WEAPON_3_1
 Plasma Cannons
 
 SHP_WEAPON_3_1_DESC
-Unlocks the Plasma Cannon, a more powerful ship's weapon than the Laser.
+Unlocks the [[shippart SR_WEAPON_3_1]], a more powerful ship's weapon than the [[shippart SR_WEAPON_2_1]].
 
 SHP_WEAPON_3_2
 Plasma Cannon 2
 
 SHP_WEAPON_3_2_DESC
-Improves Plasma Cannon by one level, increasing strength by 3.
+Upgrades [[shippart SR_WEAPON_3_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 12.
 
 SHP_WEAPON_3_3
 Plasma Cannon 3
 
 SHP_WEAPON_3_3_DESC
-Improves Plasma Cannon two levels increasing strength by 6.
+Upgrades [[shippart SR_WEAPON_3_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 15.
 
 SHP_WEAPON_3_4
 Plasma Cannon 4
 
 SHP_WEAPON_3_4_DESC
-Improves Plasma Cannon three levels increasing base strength by 9.
+The ultimate refinement of the Plasma Cannon. Upgrades [[shippart SR_WEAPON_3_1]] parts on all ships within [[metertype METER_SUPPLY]], increasing shot [[encyclopedia DAMAGE_TITLE]] to 18.
 
 SHP_WEAPON_4_1
 Death Rays
 
 SHP_WEAPON_4_1_DESC
-Unlocks the Death Ray, the ultimate ship's weapon.
+Unlocks the [[shippart SR_WEAPON_4_1]], a more powerful ship's weapon than the [[shippart SR_WEAPON_3_1]]. The ultimate ship's weapon.
```

Comment:
Realize this was from original, I'd prefer to drop use of ultimate describing this weapon, since it is surpassed by spinal cannon or flak cannon, depending on use case.